packages/email-auth/src/email-auth-server.ts

```diff
@@ -105,30 +103,35 @@ export class EmailAuthServer {
               },
             }),
           );
-
+  
           return;
         } else {
-          await this.storageAdapter.deleteOtp({
+          await storageAdapter.deleteOtp({
             otp: body.otp,
           });
         }
-
-        const token = jsonwebtoken.sign({}, this.opts?.jwtSecret as string, {
-          expiresIn: this.opts?.jwtTTL as string,
+  
+        const token = jsonwebtoken.sign({}, opts?.jwtSecret as string, {
+          expiresIn: opts?.jwtTTL as string,
           subject: user.emailAddress,
         });
-   
-        res.writeHead(200, { 'Content-Type': 'application/json' })
+  
+        res.writeHead(200, { 'Content-Type': 'application/json' });
         res.write(
           JSON.stringify({
             ok: true,
             token,
           }),
         );
         res.end();
-
+  
         return;
       }
-    });
+  
+      res.writeHead(404, { 'Content-Type': 'application/json' });
+      res.end();
```

Comment:
do not call `end` for unsupported paths, this will disable the express server

Reply:
fixed